on_boot.d/99-lego-install.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+if [ ! -f /etc/cron.d/lego ]; then
+    /mnt/data/ssl/lego.sh renew
```

Comment:
Here we should use a way to automate this /path or should it be changed

Reply:
So I only made it required to use the full path for the inclusion of the .env file, but I couldn't figure out a more elegant way to do it. As in, if you were in `/mnt/data/ssl` and ran `./lego.sh` it would include the .env fine, but if you run it from cron via a full path, it doesn't know where the .env file is to include it. I'm open to suggestions as to how to make it more elegant.

The only thing I could think of was to hard code the include path of the env and ditch that BASEDIR stuff.

Reply:
I am not used to bash... It's working, that's all matters to me 🙈

Reply:
haha ok. When I have some time this week I'll see about merging this and updating maybe moving to /mnt/data/ssl if I can figure the difference between the two mounts out.

Reply:
I think there is no difference.

Reply:
<img width="426" alt="Captura de pantalla 2020-07-07 a las 15 12 29" src="https://user-images.githubusercontent.com/1969593/86786532-087aa100-c05c-11ea-90c7-97cfd2162d26.png">

If you look the mountpoints of udm the permanent storage is mounted at `/mnt/data`.

Not sure about `/data/persistent` but has only 128 MB.